streams/src/main/java/org/apache/kafka/streams/kstream/GlobalKTable.java

```diff
@@ -19,23 +19,28 @@
 
 import org.apache.kafka.common.annotation.InterfaceStability;
 import org.apache.kafka.streams.KafkaStreams;
+import org.apache.kafka.streams.KeyValue;
+import org.apache.kafka.streams.state.ReadOnlyKeyValueStore;
 
 /**
  * {@link GlobalKTable} is an abstraction of a <i>changelog stream</i> from a primary-keyed table.
- * Each record in this stream is an update on the primary-keyed table with the record key as the primary key.
+ * Each record in this changelog stream is an update on the primary-keyed table with the record key as the primary key.
  * <p>
- * A {@link GlobalKTable} is fully replicated per {@link KafkaStreams} instance. Every partition of the underlying topic
- * is consumed by each {@link GlobalKTable}, such that the full set of data is available in every {@link KafkaStreams} instance.
- * This provides the ability to perform joins with {@link KStream}, and {@link KTable},
- * without having to repartition the input streams. All joins with the {@link GlobalKTable} require that a {@link KeyValueMapper}
- * is provided that can map from the (key, value) of the left hand side to the key of the right hand side {@link GlobalKTable}
+ * In contrast to a {@link KTable} that is partitioned over all {@link KafkaStreams} instances, a {@link GlobalKTable}
+ * is fully replicated per {@link KafkaStreams} instance.
+ * Every partition of the underlying topic is consumed by each {@link GlobalKTable}, such that the full set of data is
+ * available in every {@link KafkaStreams} instance.
+ * This provides the ability to perform joins with {@link KStream} and {@link KTable}, without having to repartition the
```

Comment:
We should actually remove the reference to `KTable` as we don't currently support it